src/functions.h

```diff
@@ -54,9 +54,15 @@
 
 typedef struct functionLibInfo functionLibInfo;
 
+/* ValkeyModule type aliases for scripting engine structs and types. */
+typedef ValkeyModuleScriptingEngineCtx EngineCtx;
+typedef ValkeyModuleScriptingEngineFunctionCtx FunctionCtx;
+typedef ValkeyModuleScriptingEngineCompiledFunction CompiledFunction;
+typedef ValkeyModuleScriptingEngineMethods EngineMethods;
```

Comment:
Nice.

Just thinking, apart from in the module API, we use a lowercase first letter for types and structs, at least in most cases (or always? I'm not sure). For consistency, it's probably best to follow the existing style.

Reply:
Yeah, you're right. I'll change it to be consistent in the rest of the code.